pipelines/_ingredients/ApPipe.yaml

```diff
@@ -158,6 +162,7 @@ tasks:
       connections.diaObjects: "{fakesType}dia_object_apdb"
       connections.newDiaSources: "{fakesType}new_dia_source"
       connections.marginalDiaSources: "{fakesType}marginal_new_dia_source"
+      idGenerator.release_id: parameters.release_id
```

Comment:
To confirm, the release ID is needed in `calibrateImage`, `detectAndMeasureDiaSource`, and `associateApdb`? Why would the last need to generate IDs?

I can't think of any tasks that might be missing, but the pipeline isn't my area of expertise.